src/components/Card/index.js

```diff
@@ -82,75 +90,146 @@ MaterialType.propTypes = {
 const CardHeader = ({response, type, expanded}) => {
   const headerComponent = cardTypes[type].headerComponent ? cardTypes[type].headerComponent(response, expanded) : null
   return (
-    <div className='w-100'>
+    <div className='flex flex-column items-center w-100'>
       {headerComponent}
     </div>
-  )  
+  )
 }
 CardHeader.propTypes = {
   response: PropTypes.object,
   type: PropTypes.string.isRequired
 }
 
+const CardBody = ({title, instructor, type}) => {
+  return (
+    <div className={`${type === 'playlist' ? 'ph4' : ''}`}>
+      <h3 className={titleHeadingClasses}>{title}</h3>
+      <div className={instructorNameClasses}>{instructor}</div>
+    </div>
+  )
+}
+CardBody.propTypes = {
+  title: PropTypes.string.isRequired,
+  instructor: PropTypes.string.isRequired,
+  type: PropTypes.oneOf(keys(cardTypes))
+}
+
 const CardFooter = ({response, type}) => {
   const metaComponent = cardTypes[type].metaComponent ? cardTypes[type].metaComponent(response) : null
   return (
     <div className={`${footerClasses} ${cardTypes[type]['footerClasses']}`}>
       {metaComponent}
       <MaterialType type={type} />
     </div>
-  ) 
+  )
 }
 CardFooter.propTypes = {
   response: PropTypes.object,
   type: PropTypes.string.isRequired
 }
 
-const CardBody = ({title, instructor}) => {
-  return (
-    <div>
-      <h3 className={titleHeadingClasses}>{title}</h3>
-      <div className={instructorNameClasses}>{instructor}</div>
-    </div>
-  )
-}
-CardBody.propTypes = {
-  title: PropTypes.string.isRequired,
-  instructor: PropTypes.string.isRequired
-}
+const StyledCardContainer = styled.div`
+  ${props => cardTypes[props.type]['cardStyles']}
+  ${props => props.expanded === 'horizontal' ? 'max-width: 760px' : ''}
```

Comment:
It seems to me some styles that could be tachyons classes like max-width are moving into `styled-components`; why not use tachyons for everything unless it isn't in tachyons, then add it to `styled-components`? So we have the consistent number system of tachyons instead of arbitrary hard-coded values. Not required for this PR but something to think about in the future.